docusaurus/docs/iOS/customization/localization.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Adding Localization
+---
+
+The SDK offers you in-house localization for all strings contained in UI components. For our internal localization, we provide `strings` file for each language that we translate. For advanced plural localization cases (typing indicator showing how many people are typing or how many responses are in thread), we use `stringsdict` with `NSStringPluralRuleType`. 
+
+## Changing the device's language
+
+Stream Chat tries to use the same language as the device it is running on. If the device is set to a language that Stream Chat does not support, then Stream Chat will default to English.
+See [how to change device language](https://support.apple.com/en-us/HT204031).
+There are however some workarounds how you can enforce the application to use only specific languages or let the users choose.
```

Comment:
If we're calling out workflows for forcing the app to only use a specific language, we should link to them or add them to this page.

Reply:
According to internal discussion, deleted the sentence. Thank you :)